From acaa00fde6eb0a26ab93696af3ac898b6110dbcf Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Fri, 7 Mar 2025 12:13:28 +0100
Subject: [PATCH 22/22] OvmfPkg/PlatformInitLib: fix typo

There is a typo in the FwCfg file name, it's spelled
'tianocore' not 'tianocode'.  Fix it.

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
(cherry picked from commit 6234d09f7582e7f759ca93c9af1583b01cb5954d)
---
 OvmfPkg/Library/PlatformInitLib/MemDetect.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/OvmfPkg/Library/PlatformInitLib/MemDetect.c b/OvmfPkg/Library/PlatformInitLib/MemDetect.c
index b6aefb321df5..1f987f2efd91 100644
--- a/OvmfPkg/Library/PlatformInitLib/MemDetect.c
+++ b/OvmfPkg/Library/PlatformInitLib/MemDetect.c
@@ -994,7 +994,7 @@ PlatformSetupPagingLevel (
   }
 
   Status = QemuFwCfgParseUint32 (
-             "opt/org.tianocode/PagingLevel",
+             "opt/org.tianocore/PagingLevel",
              FALSE,
              &PagingLevel
              );
-- 
2.48.1

